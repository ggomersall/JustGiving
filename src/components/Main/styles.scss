.donations__section.container {
  margin-top: 25px;
}