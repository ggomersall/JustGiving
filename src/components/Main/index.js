import React, {Component} from 'react';

import Charity from '../Charity';
import Donations from '../Donations';

import './styles.scss';

class Main extends Component {
  constructor(props) {
    super(props)
  }

  render() {
    const {charityData, donationsData} = this.props
    return (
      <div className="main__section main_wrapper">
        <Charity charity={charityData} />
        <Donations charity={charityData} donations={donationsData} />
      </div>
    ) 
  }
}

export default Main