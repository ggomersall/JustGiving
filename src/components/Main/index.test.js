import React from 'react';
import { shallow } from 'enzyme';
import Main from './index.js';

describe('Testing Main Component', () => {

  let wrapper;

  it('renders correctly', () => {
    shallow(<Main />);
  });

  test('it should render loading props correctly', () => {
    const likes = 100;
    const care = true;
    const charity = {data: 'ipsum data'};
    const donations = {data: 'ipsum data 2'};
    wrapper = shallow(<Main likes={likes} care={care} charity={charity} donations={donations} />);
  });

  test('Snapshot', () => expect(<Main />).toMatchSnapshot());

});

