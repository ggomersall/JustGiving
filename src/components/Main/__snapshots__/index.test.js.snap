// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Main Component Snapshot 1`] = `<Main />`;
